src/EPR.PRN.Backend.API/Controllers/PrnController.cs

```diff
@@ -61,60 +69,72 @@ public async Task<IActionResult> GetSearchPrns([FromHeader(Name = "X-EPR-ORGANIS
     [ProducesResponseType(typeof(List<PrnDto>), 200)]
     public async Task<IActionResult> GetAllPrnByOrganisationId([FromHeader(Name = "X-EPR-ORGANISATION")] Guid orgId)
     {
-        var prn = await _prnService.GetAllPrnByOrganisationId(orgId);
+        logger.LogInformation("{Logprefix}: PrnController - GetAllPrnByOrganisationId: Api Route api/v1/prn/organisation", logPrefix);
+        logger.LogInformation("{Logprefix}: PrnController - GetAllPrnByOrganisationId: request for user organisation {Organisation}", logPrefix, orgId);
 
-        return Ok(prn);
+        var prns = await prnService.GetAllPrnByOrganisationId(orgId);
+        logger.LogInformation("{Logprefix}: PrnController - GetAllPrnByOrganisationId: Prns returned {Prns}", logPrefix, JsonConvert.SerializeObject(prns));
+
+        return Ok(prns);
     }
 
     [HttpGet("obligationcalculation/{year}")]
     [ProducesResponseType(typeof(List<ObligationData>), 200)]
     [ProducesResponseType(400)]
     [ProducesResponseType(500)]
-    public async Task<IActionResult> GetObligationCalculation([FromHeader(Name = "X-EPR-ORGANISATION")] Guid organisationId,
-        [FromRoute] int year)
+    public async Task<IActionResult> GetObligationCalculation([FromHeader(Name = "X-EPR-ORGANISATION")] Guid organisationId, [FromRoute] int year)
     {
+        logger.LogInformation("{Logprefix}: PrnController - GetObligationCalculation: Api Route api/v1/prn/obligationcalculation/{Year}", logPrefix, year);
+        logger.LogInformation("{Logprefix}: PrnController - GetObligationCalculation: request to get Obligation Calculation for user organisation {Organisation} for {Year}", logPrefix, organisationId, year);
+
         if (year < _config.StartYear || year > _config.EndYear)
         {
+            logger.LogError("{Logprefix}: PrnController - GetObligationCalculation: Invalid year provided: {Year}.", logPrefix, year);
             return BadRequest($"Invalid year provided: {year}.");
         }
 
-        var obligationCalculation = await _obligationCalculatorService.GetObligationCalculation(organisationId, year);
+        var obligationCalculation = await obligationCalculatorService.GetObligationCalculation(organisationId, year);
```

Comment:
Why has this field's rename removed the underscore?

Reply:
Primary constructor didn't need additional local variable.
Since we had additional Dependency Injection, it came up with sonar suggestion